### DISCUSSION TITLE
How to countinue creating flet project

### DISCUSSION THREAD

**Discussion by SKD8** [flet-dev/flet#3933] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the requested feature

So what the documentation says 
1. Create a project folder
2. Crete new .venv( also allow script execution for current session if not why should I allow script execution all the time)
3. Activate script
5. Install flutter
6. Now you can create app
Do I have to do this everytime I will create a new project. is there a easy way to do this . Please provide proper documentation for setup with photos. Also when i continued developing my app and was activating the .venv. I got Module not found error

**Comment by tanmay-bhatgare** [comment]
What's you main problem? Creating new project every time? Describe clearly.
